node/node_handler.go

```diff
@@ -478,8 +478,8 @@ func (node *Node) AddNewBlock(newBlock *types.Block) error {
 		utils.Logger().Error().
 			Err(err).
 			Uint64("blockNum", newBlock.NumberU64()).
-			Bytes("parentHash", newBlock.Header().ParentHash.Bytes()[:]).
-			Bytes("hash", newBlock.Header().Hash().Bytes()[:]).
```

Comment:
As a side node I think `Bytes()[:]` its the same as `Bytes()` since `Bytes()` implicitly uses `len()`

Reply:
```
// Bytes gets the byte representation of the underlying hash.
func (h Hash) Bytes() []byte { return h[:] }
```